### ISSUE TITLE
Filter Text update

### ISSUE THREAD

**Issue by dnak0** [yhirose/vscode-filtertext#20]
Could you please update to 0.0.15 version on **vscode marketplace**.
package.json doesn't contain **filterPredefined** command so it can be used.
I have seen a token that you deployed with extension. You really shouldn't do that.

**Comment by yhirose** [comment]
@nakonechniyd, thanks for the reminder and the security problem. I totally forgot updating the version. What you mean by 'a token' is the '__metadata' property, right?

**Comment by dnak0** [comment]
by token, I mean file **PersonalAccessToken.txt** in extension folder
just do 
```
code --install-extension yhirose.filtertext
ls -l yhirose.filtertext-0.0.14
``` 
you will see
```
4.0K Jul 24 14:06 .
4.0K Jul 24 14:17 ..
4.0K Jul 24 14:06 images
1.1K Jul 24 14:06 LICENSE.txt
4.0K Jul 24 14:06 node_modules
4.0K Jul 24 14:06 out
3.6K Jul 24 14:27 package.json
  53 Jul 24 14:06 PersonalAccessToken.txt
2.4K Jul 24 14:06 README.md
2.3K Jul 24 14:06 .vsixmanifest
```

**Comment by yhirose** [comment]
@nakonechniyd, thanks for your help! I just published the version to 0.0.15 to the marketplace.
